src/Compilers/Core/Desktop/AnalyzerFileReference.cs

```diff
@@ -85,13 +85,27 @@ public override ImmutableArray<DiagnosticAnalyzer> GetAnalyzersForAllLanguages()
         {
             if (_lazyAllAnalyzers.IsDefault)
             {
-                var allAnalyzers = MetadataCache.GetOrCreateAnalyzersFromFile(this);
-                ImmutableInterlocked.InterlockedInitialize(ref _lazyAllAnalyzers, allAnalyzers);
+                ImmutableInterlocked.InterlockedInitialize(ref _lazyAllAnalyzers, CreateAnalyzersForAllLanguages(this));
             }
 
             return _lazyAllAnalyzers;
         }
 
+        private static ImmutableArray<DiagnosticAnalyzer> CreateAnalyzersForAllLanguages(AnalyzerFileReference reference)
+        {
+            // get all analyzers in the assembly;
```

Comment:
Please capitalize the first letter and end the comment with a period. (I wonder how the world would be different if programming language statements ended with periods.)